src/api/app/controllers/attribute_controller.rb

```diff
@@ -34,134 +36,76 @@ def index
     render xml: xml
   end
 
-  # /attribute/:namespace/_meta
-  def namespace_definition
-    if params[:namespace].nil?
-      raise MissingParameterError, "parameter 'namespace' is missing"
+  def show_namespace_definition
+    if (@an = AttribNamespace.where(name: ensure_namespace).select(:id, :name).first)
+      render template: 'attribute/namespace_definition'
+    else
+      render_error message: "Unknown attribute namespace '#{@namespace}'",
+        status: 404, errorcode: 'unknown_attribute_namespace'
     end
-    namespace = params[:namespace]
+  end
 
-    if request.get?
-      @an = AttribNamespace.where(name: namespace).select(:id, :name).first
-      unless @an
-        render_error message: "Unknown attribute namespace '#{namespace}'",
-          status: 404, errorcode: 'unknown_attribute_namespace'
-      end
-      return
-    end
+  def delete_namespace_definition
+    AttribNamespace.where(name: ensure_namespace).destroy_all
```

Comment:
Please use `.find_by_name...` (inside `require_namespace`, see above) and `.destroy`